### ISSUE TITLE
class comment markdown not shown in Pharo 10 as in Pharo 9

### ISSUE THREAD

**Issue by kasperosterbye** [guillep/pharo-core#9915]
The browser no longer show the class comment as rendered markdown, but as simple text. While this is a rather obvious, I just wanted to place it as an issue.

Pharo 9:
<img width="853" alt="image" src="https://user-images.githubusercontent.com/17723745/132284471-c4146674-93d6-408f-bea3-97c57e1242f2.png">

Pharo 10:
<img width="801" alt="image" src="https://user-images.githubusercontent.com/17723745/132284508-18fadf48-f3e4-464e-bd77-5170a18ebd18.png">

**Comment by astares** [comment]
- Microdown support was removed (although introduced in P9) for P10 which came as a surprise 
- as far as I know there will be a new Microdown with support also for external MD files for the help system

So I guess this is intended and not a bug. I guess more a problem of communication to external community what is planned and done for this iteration. Often only the internal team knows whats planned and done in advance and thats a pitty and sad. It  makes it hard to follow and contribute from the outside. 

Side note: that's not a rant - just how things look from the outside (it is often extremely time consuming to find things out even for people who know processes).

Maybe @Ducasse could comment better on the state of Microdown and (reintroducing) documentation tools

**Comment by Ducasse** [comment]
We removed it so that I can work and I cannot justify any breath I do.
[https://github.com/pharo-project/pharo/wiki/05-PhIO-MicrodownForDocs](https://github.com/pharo-project/pharo/wiki/05-PhIO-MicrodownForDocs)
Pharo 10 is alpha. 


<blockquote><img src="https://repository-images.githubusercontent.com/64123480/451beb80-d08b-11e9-9ade-21d166d787d9" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/pharo-project/pharo">05 PhIO MicrodownForDocs · pharo-project/pharo Wiki</a></strong></div><div>Pharo is a dynamic reflective pure object-oriented language supporting live programming inspired by Smalltalk. - 05 PhIO MicrodownForDocs · pharo-project/pharo Wiki</div></blockquote>

**Comment by Ducasse** [comment]
If you want to know what is happening in Microdown please look at the kanban there. 
I'm working hope to finish this evening on a new path resolver.

**Comment by kasperosterbye** [comment]
Super - thanks for the link, and I look forward to take a look at the status.

**Comment by Ducasse** [comment]
Well this is not as advanced I would love. Now the dev branch is broken because I'm refactoring the path resolution. I hope to get done soon.
